config.yml

```diff
@@ -1,41 +0,0 @@
----
```

Comment:
I don't like the idea of removing this file.
I believe you should keep it and ask the user whether or not you import the prices from the config file or hit the API.